### PR TITLE
Misc changes 

### PR DESCRIPTION
**Pull request by amymok**
### Additions
- Add title sentence above outputs

### Removals
- Remove intro paragraph

### Changes
- Update DP Scenario Intro
- Reformat the indentation

### Review
@cfarm @virginiacc

### CODE REVIEW

**Comment by amymok** [comment]
### Remove Intro Paragraph:
  * Desktop:
screen shot 2015-05-29 at 12 14 40 pm
  * Mobile:
screen shot 2015-05-29 at 1 45 31 pm

### Update DP Scenario Intro:
  * Desktop:
screen shot 2015-05-29 at 1 44 34 pm
  * Mobile:
screen shot 2015-05-29 at 1 45 46 pm

### Add title sentence above outputs:
  * Desktop:
screen shot 2015-05-29 at 1 44 55 pm
* Mobile:
screen shot 2015-05-29 at 1 46 03 pm

**Comment by huetingj** [comment]
Thank you for making these changes, Amy! 

<h2>Intro area</h2>
<h4>Remove intro paragraph.</h4>
Sorry to essentially take a step back, but we're now considering including a sentence beneath "Explore how loan options affect costs." Along the lines of this:

screen shot 2015-05-29 at 2 20 12 pm

I will be finishing style exploration (mostly regarding font sizes) today. The designers will then come to a decision about it, and I'll document that decision/the right font sizes in this issue. Thanks and sorry again for that change!

Here are some other thoughts:

<h4>There should not be a green line under the breadcrumb.</h4>

screen shot 2015-05-29 at 2 22 46 pm

The breadcrumb should be 30 px above the title like on the rate checker page:
screen shot 2015-05-29 at 2 23 52 pm

<h4>"Explore how loan options affect costs." should be on mobile.</h4>

screen shot 2015-05-29 at 2 54 19 pm

Right now, it shows on desktop but drops off on mobile. But perhaps hold off on making this change until we get back to you re: the supporting intro sentence.

<h4>Illustration on mobile</h4>
We haven't been including the illustration on mobile. @sonnakim @stephanieosan ok with dropping it?

<h4>The size of the text in the dropdown is small in comparison to the other text.</h4>
Is this our standard for drop downs? Tricky. Can we make it bigger to match the size of the surrounding text?

screen shot 2015-05-29 at 2 29 53 pm

<h2>Title sentence above outputs</h2>
The spacing should be 60 px beneath the input section and 30 px above output section on desktop.

screen shot 2015-05-29 at 2 39 32 pm

Looks like it's currently 60 px above and below? Just checking. 

On mobile, can we have less indent space, please? Right now:
screen shot 2015-05-29 at 2 49 31 pm

It should be indented 15 px from left-hand side.

<h2> Funky stuff just to note</h2>
Hmm!
screen shot 2015-05-29 at 2 32 45 pm

screen shot 2015-05-29 at 2 34 45 pm

Let me know if any of this is mumble jumbled confusing. And I'll post soon regarding the intro title/sentence. Thank you, Amy!!

**Comment by stephanieosan** [comment]
@huetingj I think we should note these issues elsewhere. We'll want to do a thorough review of what css tweaks are necessary overall.

**Comment by huetingj** [comment]
Ok! Legit.

**Comment by stephanieosan** [comment]
I want to call out the two things that Jenn mentioned that we **should** fix before we merge this PR:

1. Getting rid of that green line at the top of the page
2. Currently the title of the tool "Explore..." is getting hidden on mobile, and we don't want that.

The rest we can port over into a github issue about css/design tweaks.

**Comment by amymok** [comment]
@cfarm @virginiacc 

I fixed the 2 items @stephanieosan @huetingj pointed out.  Please check to make sure my changes don't affect other parts of the site.

**Comment by amymok** [comment]
@emilievm @huetingj @stephanieosan @sonnakim 

Updated screenshots:

Hmm not sure if the mobile one is okay, no spacing...?

Desktop:
screen shot 2015-05-29 at 7 46 14 pm

Mobile:
screen shot 2015-05-29 at 7 46 44 pm

**Comment by stephanieosan** [comment]
Since we're not testing mobile in this upcoming round of testing, it probably makes sense to let it go for now, then come back to it when we're doing overall css tweaks. That way you can move on to other stuff :)

**Comment by huetingj** [comment]
:+1: looks/sounds good!

File: src/static/css/module/loan-comparison.less [hunk 1]
```diff
@@ -801,8 +805,7 @@
   .respond-to-min(@tablet-min, {
     .block;
     padding-top: unit(30px / @base-font-size-px, em);
-    margin-top: unit(30px / @base-font-size-px, em);
-    border-top: 2px solid @green;
+    margin-top: 0px; //unit(30px / @base-font-size-px, em);
```
Comment (cfarm) [inline src/static/css/module/loan-comparison.less]: looks like this comment can be deleted

File: src/static/js/modules/loan-comparison/components/app.js [hunk 2]
```diff
@@ -13,71 +13,75 @@ require('tooltips');
...
+        return (
+          <div>
+            
+            <div>
+                <ScenarioSection scenario={this.state.scenario}/>
+                <div className="block block__border-top block__padded-top" id="loans-container">
+                    <ScenarioHeader scenario={this.state.scenario}/>
+                    <div className="content-l">
+                        <div className="content-l_col content-l_col-3-4">
+                            <div>
```
Comment (cfarm) [inline src/static/js/modules/loan-comparison/components/app.js]: can probably remove this div around `LoanOutputTableMobileGroup`

File: src/static/js/modules/loan-comparison/components/scenario-section.js [hunk 3]
```diff
@@ -7,10 +7,13 @@ var ScenarioSection = React.createClass({
         return (
             <div id="scenario-section">
                 <div className="content-l">
-                    <div className="content-l_col content-l_col-1">
-                        <h3 className="u-hide-on-mobile">{"Let's get started."}</h3>
+                    <div className="content-l_col content-l_col-3-4">
+                        <h3>{"Explore how loan options affect costs."}</h3>
```
Comment (cfarm) [inline src/static/js/modules/loan-comparison/components/scenario-section.js]: is this how you write text in JSX? amazing

**Comment by cfarm** [comment]
I noticed a couple minor cleanup things, otherwise should be good to go.

**Comment by cfarm** [comment]
:+1:

**Pull request merged**